public/locale/en.json

```
@@ -1836,9 +1836,11 @@
   "no_appointments": "No appointments found",
   "no_attachments_found": "This communication has no attachments.",
   "no_availabilities_yet": "No availabilities yet",
+  "no_available_beds_found": "No Available Beds Found",
```

```suggestion
  "no_available_beds_found": "No available beds found",
```

Lets try to keep the only the first letter of the sentence capital and rest small, If you want all the first letters of the word in a sentence to be in caps, consider using `capitalize` tailwind class name.

---

oh ok , no let's just go for lowercase